lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -242,10 +242,10 @@ describe('GatherRunner', function() {
     });
   });
 
-  it('clears the network cache and origin storage', () => {
+  it('clears origin storage', () => {
```

Comment:
it doesnt go off and on, actually.  in the ended didnt go with what the ticket explained. :)

just a one time "Clear everything" at the start of the pass.

added test for that.